addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -131,7 +131,7 @@ function ComposerFactory({ Entity }) {
                     subtype_id,
                     subtype_xmlid: isLog ? 'mail.mt_note' : 'mail.mt_comment',
                 });
-                const messageId = await this.env.rpc({
+                messageId = await this.env.rpc({
```

Comment:
What happened before? Was the variable shadowed inside the `else`, or isn't it a syntax error? We didn't see a crash?

Reply:
`messageId` was undefined below so I think the const implied a new scoped variable and did not update the "global" one

Reply:
Yep confirmed behavior of `let`/`const`, this doesn't work the same as `var` did.